netts/version.py

```diff
@@ -1 +1,8 @@
-__version__ = "0.1.0"
+try:
+    from importlib.metadata import version  # type: ignore
+
+except ImportError:
+    from importlib_metadata import version
+
+
+__version__ = version("netts")
```

Comment:
Does this mean we only need to change it in pyproject.toml from now on?

Reply:
exactly